datadog/resource_datadog_monitor.go

```diff
@@ -84,7 +84,7 @@ func resourceDatadogMonitor() *schema.Resource {
 				},
 				"priority": {
 					Description: "Integer from 1 (high) to 5 (low) indicating alert severity.",
```

Comment:
```suggestion
					Description: "Integer from 0 (high) to 5 (low) indicating alert severity.",
```
Should this be updated to say 0 - 5?

Reply:
I had the same inclination as you did however I spoke with PM about this and they requested to not change the documentation as this is a private beta feature.